tests/subscriptions/test_worker.py

```diff
@@ -244,6 +244,7 @@ def test_subscription_worker(
         future_result = request, result = future.result()
         assert message.payload.task.timestamp == timestamp
         assert message.payload == SubscriptionTaskResult(task, future_result)
+        del result["profile"]
```

Comment:
why did this test need to be modified?

Reply:
Because it checks the result exactly on line 270. Although I had to change it there anyways because there's also trace output now.